rtf_vis_tool/src/Components/PCViewer.js

```diff
@@ -63,16 +63,16 @@ function PCViewer(props) {
 
         /* Variable arr_points is an (N x 6) array, with the first 3 entries as (x,y,z) and the last
            3 entries as (R,G,B). */
-        const arr_points = arrStringPoints.map(point => point.split(" ").map(Number));
+        const arr_points = arrStringPoints.map(point => point.split(" ").map(Number).slice(1,7));
```

Comment:
why is this needed now?

Reply:
Because before, the array `arr_points` was actually not of length 6. It contained all fields of information shows in the picture below. But now, I've decided to only focus on indices 1 through 7, which are x,y,z and r,g,b.

![image](https://user-images.githubusercontent.com/40643713/116263092-16aec080-a747-11eb-90ef-3e7b1d46c445.png)